README.md

```diff
@@ -113,11 +113,18 @@ sudo chown root:root /etc/apt/sources.list.d/microsoft-prod.list
 
 2. Install
 
+##### v2
 ```bash
 sudo apt-get update
 sudo apt-get install azure-functions-core-tools
```

Comment:
I will also add a package `azure-functions-core-tools` as part of my release pipeline updates here - https://github.com/Azure/azure-functions-core-tools/issues/1859

Once I do that, I will update this to be `azure-functions-core-tools-2`